packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -50,6 +50,69 @@ export const EXT_OVERRIDES_DIR =
 export const EXT_OVERRIDES_DIR_NO_SLASH = EXT_OVERRIDES_DIR?.replace(/^\//, '')
 export const EXT_EXTENDS = pkg?.ccExtensibility?.extends
 export const EXT_EXTENDABLE = pkg?.ccExtensibility?.extendable
+// in a Template Extensibility project, these can include BOTH the node_modules/*
+// AND the node_modules/<extends>/node_modules/* npm deps, so we route here to
+// prevent duplication in the final bundle. We want a better way of doing this,
+// but for now this is the best option we've found.
+
+// TODO: can these be handled in package.json as peerDependencies?
+// https://salesforce-internal.slack.com/archives/C0DKK1FJS/p1672939909212589
+
+export const OVERRIDABLE_DEPS = [
+    '@chakra-ui/icons',
+    '@chakra-ui/react',
+    '@chakra-ui/skip-nav',
+    '@chakra-ui/system',
+    '@chakra-ui/theme',
```

Comment:
I was only able to find `chakra-ui/layout/dist` as duplicated and it's minor, around 3 kB Gzipped.
![client](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/6024102/2fbd7159-7a29-442e-86bc-aa5f68261855)

Reply:
Yeah so @kevinxh this brings up the question of transitive dependencies. Not sure what to do about this just yet. Let's discuss in the morning, me, you, @adamraya

Reply:
(sent an invite for tomorrow)

Reply:
Duplicates only happen from the monorepo, this is no longer relevant, as customer projects won't have this issue